yt/visualization/volume_rendering/scene.py

```diff
@@ -302,7 +304,28 @@ def save(self, fname=None, sigma_clip=None):
         self.render()
 
         mylog.info("Saving render %s", fname)
-        self._last_render.write_png(fname, sigma_clip=sigma_clip)
+        # We can render pngs natively but for other formats we defer to
+        # matplotlib.
+        if suffix == '.png':
+            self._last_render.write_png(fname, sigma_clip=sigma_clip)
+        else:
+            from matplotlib import pyplot as plt
+            fig, ax = plt.subplots()
+            shape = self._last_render.shape
+            fig.set_size_inches(shape[0]/100., shape[1]/100.)
+            fig.subplots_adjust(left=0, bottom=0, right=1, top=1)
+            ax.set_axis_off()
+            out = self._last_render
+            nz = out[:, :, :3][out[:, :, :3].nonzero()]
+            max_val = nz.mean() + sigma_clip * nz.std()
+            alpha = 255 * out[:, :, 3].astype('uint8')
+            out = np.clip(out[:, :, :3] / max_val, 0.0, 1.0) * 255
+            out = np.concatenate(
+                [out.astype('uint8'), alpha[..., None]], axis=-1)
+            # not sure why we need rot90, but this makes the orentation
```

Comment:
Yup. For testing I was using this script:

```
import yt

ds = yt.load('IsolatedGalaxy/galaxy0030/galaxy0030')

sc = yt.create_scene(ds, 'density', lens_type='perspective')

for ext in ['.pdf', '.png']:
    sc.save('rendering{}'.format(ext), sigma_clip=6.0)
```

Which produced the following two files:

![rendering](https://user-images.githubusercontent.com/3126246/28536524-82ad4b68-706d-11e7-8c3c-d35e3d496792.png)
[rendering.pdf](https://github.com/yt-project/yt/files/1170675/rendering.pdf)